config/settings.yml

```diff
@@ -83,9 +83,12 @@
         :poll: 15.seconds
       :event_catcher_ibm_cloud_iks:
         :poll: 15.seconds
+      :event_catcher_prometheus:
+        :poll: 20.seconds
     :queue_worker_base:
       :ems_metrics_collector_worker:
         :ems_metrics_collector_worker_vpc: {}
+        :ems_metrics_collector_worker_ibm_cloud: {}
```

Comment:
Earlier I had `ems_metrics_collector_worker_iks` here but the Travis build failed and suggested `ems_metrics_collector_worker_ibm_cloud` instead. How is this worker name determined?

Reply:
You need to override the settings_name method in that worker, the default settings_name in `MiqWorker` base doesn't handle child managers well.

E.g. from the IbmCloud::ContainerManager::RefreshWorker:
```
  def self.settings_name
    :ems_refresh_worker_ibm_cloud_iks
  end 
```

Recommend staying with this naming convention and using `:ems_metrics_collector_worker_ibm_cloud_iks`